R/ResampleResult.R

```diff
@@ -67,6 +67,8 @@ autoplot.ResampleResult = function(object, type = "boxplot", measure = NULL, ...
       autoplot(precrec::evalmod(as_precrec(object)), curvetype = "prc", show_cb = TRUE)
     },
 
+    "prediction" = plot_learner_prediction_resample_result(object$learners[[1]], object, measure, ...),
```

Comment:
Ugh, the name. Can we just call it `plot_learner_prediction()`, too?

Reply:
1. The name is only internal.
2. `plot_learner_prediction()` has the arguments `learner` (without model), `task` ... and `plot_learner_prediction_resample_result()` has `learner` (with model), `object` (ResampleResult), ... . We could get rid of the ResampleResult Object if we decide to pass the fold argument and the tain and test ids (if we want to differentiate).